{{cookiecutter.repo_name}}/{{cookiecutter.app_name}}/models.py

```diff
@@ -1,10 +1,7 @@
 # -*- coding: utf-8 -*-
 """
 Database models for {{cookiecutter.app_name}}.
-"""
-
-from __future__ import absolute_import, unicode_literals
-{% if cookiecutter.models != "Comma-separated list of models" %}
+"""{% if cookiecutter.models != "Comma-separated list of models" %}
```

Comment:
Since these are Jinja2 templates, we can use `{%-` and `-%}` where needed to control the whitespace instead of awkward juxtapositions like this.

Reply:
Ah, good to know.  Not sure I want to tinker with this again right now since I just got it working, but will definitely keep it in mind if I need to come back and edit this file again.